torch/onnx/_internal/docs/gpt.py

```diff
@@ -113,24 +114,28 @@ def test_gpt2_one_shot(model_name):
     onnx_model = export_without_kwargs(model, **inputs, use_binary_format=True)
 
     ref_outputs, _ = tree_flatten(model(**inputs, return_dict=False))
-    pth_outputs = run_ort(
+    ort_outputs = run_ort(
         onnx_model, onnx_model_text, (input_ids, attention_mask), ref_outputs
     )
-    for _1, _2 in zip(ref_outputs, pth_outputs):
-        print(_1 - _2)
-        assert torch.allclose(_1, _2)
+    for _1, _2 in zip(ref_outputs, ort_outputs):
```

Comment:
I would take the opportunity to rename this `_1` `_2` thing. ref_output, ort_output seem reasonable